src/Mizar/Mizar.cpp

```diff
@@ -28,15 +29,24 @@
 
 int main(int argc, char* argv[]) {
   // The main in its current state is used to testing/experimenting and serves no other purpose
-  if (argc < 3) {
-    ORBIT_ERROR("Two filepaths should be given");
+  if (argc < 7) {
```

Comment:
I guess this is just a test main but with so many options consider Abseilf flags.

Reply:
So many arguments are not here for long. They will not be needed after the first steps towards the UI